src/applications/simple-forms/40-0247/components/SupportingDocsViewField.jsx

```diff
@@ -0,0 +1,51 @@
+import React from 'react';
+
+import PropTypes from 'prop-types';
+
+import { displayFileSize } from 'platform/utilities/ui/index';
+
+import { supportingDocsDescription } from '../helpers';
+
+const SupportingDocsViewField = props => {
+  const { defaultEditButton, formData } = props;
```

Comment:
I don't understand where defaultEditButton comes from. how is that hooked up?

Reply:
defaultEditButton's provided as a param by forms-lib to uiSchema 'ui:objectViewField' prop function.  This particular function-param's not documented; I had to console-log props to find it.